### ISSUE TITLE
Violation from info add on

### ISSUE THREAD

**Issue by thearegee** [storybooks/storybook-addon-a11y#11]
Hello,

Using a11y addon with info addon causes a violation as it tries to validate the color contrast of the show info toggle.

See attached image:

![screenshot](https://cloud.githubusercontent.com/assets/8751165/26722929/080ebff6-478a-11e7-9e24-91f6ab4d376c.png)

Is there a work around for this?

Thanks,
Rob

**Comment by ndelangen** [comment]
Hi @thearegee Sorry, this will not happen soon, likely both addons will need refactoring to fix this, possibly with api changes.

**Comment by jbovenschen** [comment]
This is most likely an issue with the order of both decorators, if this addon is loaded before the info-addon it will try to validate the components inside that also. 

If we want to fix this we should have a way to get a reference to the story, don't know if anything like that is viable inside storybook @ndelangen?

**Comment by ndelangen** [comment]
What is planned is to have info render in a panel instead of the preview.

**Comment by chrisdhanaraj** [comment]
@thearegee easiest way for now is just override the color to something accessible

```
#root > div > div > a[role="button"] {
  background-color: #3d70b2 !important;
}
```

is what we're doing

**Comment by ndelangen** [comment]
FYI I have merged this repo into the monorepo!

Please do not open any issues or PR's directly on this repo, but instead open them here:
https://github.com/storybooks/storybook
